scripts/functional-tests.sh

```diff
@@ -595,6 +524,89 @@ test_server_chat_template() {
     test_server_template_value "$SCRIPTDIR/test-data/mock-template.txt" 3
 }
 
+#########
+# SETUP #
+#########
+set -ex
```

Comment:
Why use -x here?

Reply:
It's a bit unusual here, I'd keep built-in `set` and `export` at the beginning of the file. Or perhaps the entire SETUP section.

Reply:
`set` and `export` at the top :)

Reply:
Will do!